src/main/resources/react4xp/components/parts/ChildList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './ChildList.module.css';
+import { LinkComponent, Part } from "@enonic/react-components";
+import { Link } from "@enonic/react-components";
+
+export const ChildList = (props) => {
+    const { descriptor, type, mode, path, componentRegistry, component, result, content, params, total, names, paths, ...partProps } = props;
```

Comment:
You define lots of properties that you don't use just to have `...partProps` left, but you could instead define partProps with just the needed fields.